setup.cfg

```
@@ -37,7 +37,7 @@ python_requires = >=3.6
 install_requires =
     binary
     click
-    google-cloud-bigquery >= 0.29.0
+    google-cloud-bigquery >= 2.11.0
```

google-cloud-bigquery 2.30.0 has just been released supporting Python 3.10:

* https://pypi.org/project/google-cloud-bigquery/2.30.0/

```suggestion
    google-cloud-bigquery >= 2.30.0
```

---

I don't think we need 2.30.0:

- In Python 3.9 we can use >=2.11.0.
- In Python 3.10 pip will choose 2.30.0 as it have no choice.

So `>= 2.11.0` looks good to me.

---

We could run two tests in tox, one with the minimum version, and one with the maximum version of out dependencies, to ensure it works, like I'm trying to do here: https://github.com/JulienPalard/oeis/blob/7d6c314c02983c7a47f901d07d0fffc8b772155a/tox.ini#L36;L39 ? Maybe in another PR.

---

Ah good point that 3.10 will pick the right one. I think we'll be fine keeping it simpler and testing just a single version. Thanks!